superset/config.py

```diff
@@ -699,6 +700,7 @@ class CeleryConfig(object):
 # a dedicated period of time to allow chart producers to update their charts before
 # mass migrating all charts to use the [start, end) interval.
 SIP_15_ENABLED = False
+SIP_15_GRACE_PERIOD_END: Optional[date] = None  # exclusive
 SIP_15_DEFAULT_TIME_RANGE_ENDPOINTS = ["unknown", "inclusive"]
 SIP_15_TOAST_MESSAGE = 'Action Required: Preview then save your chart using the new time range endpoints <a href="{url}" class="alert-link">here</a>.'
```

Comment:
should we add the grace period to the toast to say when the action is required by?

Reply:
Agreed.

Reply:
@etr2460 given that the grace period is deployment specific, individual deployments should configure their messaging accordingly.